app/models/course/condition.rb

```diff
@@ -5,4 +5,9 @@ class Course::Condition < ActiveRecord::Base
   belongs_to :conditional, polymorphic: true
 
   delegate :satisfied_by?, to: :actable
+
+  ALL_CONDITIONS = [
+    Course::Condition::Achievement.class_name.downcase,
```

Comment:
Your suggestion to use `url_for` works! I updated that part but forgot to refactor this accordingly.